pallets/delegation/Cargo.toml

```diff
@@ -28,9 +27,9 @@ sp-keystore = {branch = "polkadot-v0.9.13", default-features = false, git = "htt
 
 [dependencies]
 # Internal dependencies
+attestation = {default-features = false, path = "../attestation"}
```

Comment:
the dependency can never have existed since we already had the `attestation -> delegation` dependency!?
The longer i think about it, the more i feel like it should go into runtime-common.

Reply:
on the other hand we already have the link to the attestation pallet because of the attest permission. So even if it's not set in code, the dependency is there. So why should we not make this official by having a dependency to the attestation crate.